ext/merge2std2/iterators.tex

```diff
@@ -1034,11 +1034,12 @@
     StrictWeakOrder<F&, reference_t<I1>, reference_t<I2>> &&
     StrictWeakOrder<F&, iter_common_reference_t<I1>, iter_common_reference_t<I2>>;
 
-  template <class> struct indirect_result@\oldtxt{_of}@ { };
+  template <class@\newtxt{, class...}@>
+  struct indirect_result@\oldtxt{_of}@ { };
 
   template <class F, class... Is>
-    requires Invocable<F, reference_t<Is>...>
-  struct indirect_result@\oldtxt{_of}@<F(Is...)> :
+    requires @\newtxt{(Readable<Is> \&\& ...) \&\&}@ Invocable<F, reference_t<Is>...>
+  struct indirect_result@\oldtxt{_of<F(Is...)>}@ :
```

Comment:
Nope, should be: `struct indirect_result@\oldtxt{_of<F(Is...)>}\newtxt{<F, Is...>}@`. Below, also.